docs/proposals/proposal-001-trigger-and-deploy.md

```diff
@@ -112,7 +112,62 @@ removed. In future we could create nodes on demand and delete on completion.
 
 ## Design Details
 
-![Pipeline Design](./files/trigger-deploy-pipeline.svg)
+```mermaid
+---
+title: Porposal 001 Triggr & Deploy 
+---
+stateDiagram-v2
+    
+    glreleases: GetLatestReleases()
+    projdispatch: DispatchProjects()
+    K8sCluster: Equinix K8s Cluster (k3s)
+
+    state "GH Workflow Falco" as falco_pipeline {
+        falco_installManifests: DeployManifests()
+        falco_destroyManifests: DeleteResources()
+        falco_s_benchmarking: StartBenchmarking()
```

Comment:
What's going on with the casing? 😄 

Since the diagram is code lets aim for consistency. 🤓  Since Mermaid is JS should we use camel case for all the variable names?

Reply:
okay 😄